wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -424,9 +436,33 @@ public void stopWorkspace(String workspaceId, Map<String, String> options)
               if (workspace.isTemporary()) {
                 removeWorkspaceQuietly(workspace.getId());
               }
+              if (!runtimes.isAnyActive()) {
```

Comment:
so it records stopped time only if all workspaces are stopped? Shouldn't it be all user's workspaces?

Reply:
sorry, but i don't get you question

Reply:
looks like you're checking here if any workspace is active. I thought that async pod is per user, so I would assume here check if any user's workspace is active.

Reply:
You are right, thanks